### DISCUSSION TITLE
인턴 합격 후기

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#542] [4. 기타 글 모음]
안녕하세요! 정말 가고 싶었던 회사에 합격하게 되었습니다.

보통 시상식 할 때 감사했던 분들의 이름을 부르는데 왜 그런지 알 것 같습니다..ㅎㅎ 다들 정말 감사드립니다

[후기 글](https://02ggang9.github.io/etc/Sendy1/)

**Comment by stopmin** [comment]
축하헤욤 수빈 ~~!!!
그동안 너무 오고싶어했던걸 알아서 다른 사람들보다 더 응원했습니다!! 센디에서 얻고싶었던 것 배우고 싶었던 것, 느끼고 싶었던 것을 꼭 얻어갔으면 좋겠습니당😄

저도 괜사리 더 남고싶어지는...

같이 팟팅하자!!

**Reply by 02ggang9** [reply]
감사합니다 ㅎ.ㅎ

고민하고 계신 일 잘 풀리셨으면 좋겠어요~

**Comment by shkisme** [comment]
이럴수가... 😳 저에겐 정말 너무 과분한 말들을 많이 해주셨네요!!! 정말 고맙습니다 🙇 

사실은 작년 여름방학때, 거의 혼자서 개발 & 배포 작업을 끝내야 한다는 생각에 너무 힘들었었는데 수빈의 존재가 저에게는 정말 너무 큰 힘이 되었었습니다... 저로 인해 수빈님의 진로가 터닝 포인트가 되었다는 말은 들을 때 마다 기분이 좋네요 ㅎㅎㅎㅎ (나로 인해 이런 대단한 개발자가 탄생... ㄷㄷ 나 칭찬함)

개발 경험이 전무하실 때 혼자 높은 절벽을 등반하듯이 척척 오르시면서 가파르게 성장하는 모습이 되게 인상적이었는데, 그런 모습 보면서 저도 많이 배웠던 기억이 나네용 ㅎㅎ

저는 "함께 일 하고 싶은 사람"이 최고의 칭찬이라 생각하는데, 너무 고맙습니다!! 수빈도 모두가 함께 하고 싶어하는 좋은 개발자라고 생각합니다 ㅎㅎ

합격 정말 축하하고! 앞으로도 세차새차, 센디 프로젝트 같이 화이팅해서 잘 해봅시다 ~~ ! 😊

**Reply by 02ggang9** [reply]
아이고~~ 감사합니다 감사합니다!!!

두 프로젝트 화이팅 해봅시다~~ 굳굳!! ㅎㅎㅎㅎ

**Comment by gusah009** [comment]
@02ggang9 첫 면접에서 좋은 경험을 얻으신 것 같아서 제가 다 기쁘네요 ㅎㅎ

새로운 회사에서도 좋은 경험 많이 쌓고 가셨으면 좋겠습니다~!

**Reply by 02ggang9** [reply]
감사합니다 감사합니다 선배님 ㅠ.ㅠ

모의면접 정말 감사했습니다~~~ bbbbb 항상 많이 배워갑니다!!!

**Comment by pykido** [comment]
축하드립니다!~!!!

**Reply by 02ggang9** [reply]
감사합니다~~ 아트님!!

블로그 글 잘 보고 있습니다~~ 카테캠도 화이팅입니다!!

**Comment by 1jeongg** [comment]
축하드립니다~!! 앞으로도 좋은 일만 가득하시길 바랄게요 🍀

저도 센디 지원했다가 떨어졌는데 수빈님 글보니까 어떤 점이 부족했는지 알 수 있었던 것 같아요! 감사합니다 :)

방학동안 열심히 공부해서 2학기에 다시 지원하겠습니다 ㅎㅎ 🥲

현경이 통해서 엄청 잘한다는 얘기 많이 들었는데 나중에 기회가 되면 같이 밥이라도 먹으면서 친해지고 싶네요!

**Reply by 02ggang9** [reply]
감사합니다!! 원정님도 좋은 일만 가득하실 바래요~!!

진짜 꼭 2학기 때 같이 일 해보고 싶어요!! 나중에 밥 한번 먹어용~
